org.cockpit_project.CockpitClient.releases.xml

```diff
@@ -1,6 +1,15 @@
 <?xml version='1.0' encoding='utf-8'?>
 <component>
   <releases>
+    <release version="279" date="2022-1107">
```

Comment:
@allisonkarlitskaya : I suppose it complains about the missing dash here? everything else LGTM.

Reply:
I force-pushed the missing dash. Of course we need to fix that logic somewhere, but I'd like to see in the meantime if it is happy otherwise.